lib/story.dart

```diff
@@ -137,16 +102,14 @@ class Chapter {
 
   final Story story;
 
-  Chapter(this.name, this._buildFn, this.story) {
-    ctx = DashbookContext();
-  }
+  Chapter(this.name, this._buildFn, this.story) : ctx = DashbookContext();
 
   Widget widget(BoxConstraints constraints) {
     ctx.constraints = constraints;
     final Widget w = _buildFn(ctx);
 
     if (story._decorator != null) {
-      return story._decorator.decorate(w);
+      return story._decorator!.decorate(w);
```

Comment:
I know this if was not written by you, but lets change it to use the elvis operator as well

Reply:
Sure. It looks like I don't see elvis possible for a void method call. Interesting observation about myself:)